### ISSUE TITLE
Refactor detection for closed plugins

### ISSUE THREAD

**Issue by Zodiac1978** [roytanck/rt-plugin-report#58]
It looks like, that the closed plugin status is now available via API:
https://meta.trac.wordpress.org/ticket/7057#comment:12

Maybe this is more reliable for the future. We could also wait for the SVN check to break until this is refactored.

So just a heads up, that there seems to be better way now.

**Comment by Zodiac1978** [comment]
Related: https://github.com/WordPress/wporg-experiments-plugin

**Comment by apermo** [comment]
@Zodiac1978 I think #81 and/or #86 solve this.

**Comment by Zodiac1978** [comment]
@apermo I don't think it does. The API 1.2 offers much more, like the reason for closing the plugin, which could be displayed additionally.
